src/org/openbravo/erpCommon/ad_callouts/SE_Order_Organization.java

```diff
@@ -53,42 +58,48 @@ protected void execute(CalloutInfo info) throws ServletException {
         FieldProvider[] td = null;
 
         final String strOrgId = info.getStringParameter("inpadOrgId", IsIDFilter.instance);
-        final String strBPartnerId = info.getStringParameter("inpcBpartnerId",
-                IsIDFilter.instance);
-        final String strBPartnerLocationId = info.getStringParameter(
-                "inpcBpartnerLocationId", IsIDFilter.instance);
 
-        info.addResult("inpiscashvat",
```

Comment:
Is it correct to remove all this? This is part of the location setting

Reply:
Actually these settings were not being called anywhere, and they were giving me problems before, but the way the code was refactored, I was able to add them.